DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factories/DicomProcessorFactory.cs

```diff
@@ -22,8 +28,16 @@ public IAnonymizerProcessor CreateProcessor(string method, JObject settingObject
                 "remove" => new RemoveProcessor(),
                 "refreshuid" => new RefreshUIDProcessor(),
                 "keep" => new KeepProcessor(),
-                _ => null,
+                _ => _customProcessors.GetValueOrDefault(method)
```

Comment:
I think it's a factory class, which should create new instance of processor. 
and it seems may have name conflicts between built-in and customer processor.